src/assets/style/command.css

```diff
@@ -1,6 +1,5 @@
-
 #command {
-  background-color: rgba(0,0,0,0.7);
+  background-color: #141414;
```

Comment:
That is correct, you _can_ do that but that doesn't mean you _should_. These kinds of theme things are much easier to maintain by using CSS classes.